### PR TITLE
Rename to sdic + added more docs before release

### PR DESCRIPTION
**Pull request by lra**
https://app.asana.com/0/368954819476/141148480784717

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,131 @@
...
 
-Asynchronous soft constraints executed against you databases.
-Queries that are intended to be ran here should produce `bad data`,
-or data that should not be in the table that is the object of the query.
+[![CircleCI](https://circleci.com/gh/percolate/sdic.svg?style=svg)](https://circleci.com/gh/percolate/sdic)
+[![codecov](https://codecov.io/gh/percolate/sdic/branch/master/graph/badge.svg)](https://codecov.io/gh/percolate/sdic)
+
+## One line purpose
+
+`sdic` executes all the SQL queries found in a folder and display its output.
```
Comment (mpavlisin) [inline README.md]: displays

File: README.md [hunk 2]
```diff
@@ -1,42 +1,131 @@
-# sql-data-integrity-checker
+# sdic
```
Comment (mpavlisin) [inline README.md:1]: why the acronym? I know you mentioned what it means, but isnt it more obvious and clear having `sql-data-integrity-checker`?

File: README.md [hunk 3]
```diff
@@ -1,42 +1,131 @@
...
-Queries that are intended to be ran here should produce `bad data`,
-or data that should not be in the table that is the object of the query.
+[![CircleCI](https://circleci.com/gh/percolate/sdic.svg?style=svg)](https://circleci.com/gh/percolate/sdic)
+[![codecov](https://codecov.io/gh/percolate/sdic/branch/master/graph/badge.svg)](https://codecov.io/gh/percolate/sdic)
+
+## One line purpose
+
+`sdic` executes all the SQL queries found in a folder and display its output.
+
+## More detailled purpose
```
Comment (mpavlisin) [inline README.md]: detailed

File: README.md (hunk 2 above)
Comment (lra) [inline README.md:1]: it's a pain to handle `sql-data-integrity-checker` and `sql_data_integrity_checker` depending on where it's called, and `sdic` is free on pypi so... ;)

File: README.md [hunk 4]
```diff
@@ -1,42 +1,131 @@
...
+
+1. It's easy to implement
+1. It's cheap for the database to check on every change
+
+But for more complex constraints that you'd like to set, it'd be either very
+expensinve to check on every write, or even impossible to write as a
+constraint.
+
+With `sdic`, you can write you complex constraints as simple queries, and have
+the database run them asynchronously at the occurence you want.
```
Comment (mpavlisin) [inline README.md]: occurrence

File: README.md (hunk 2 above)
Comment (mpavlisin) [inline README.md:1]: ok, your call

File: README.md [hunk 5]
```diff
@@ -1,42 +1,131 @@
...
+## Example
+
+Let's say that you have a `users` table, defined like this:
+
+- `id` Primary Key `NOT NULL`
+- `firstname` `NULL`
+- `lastname` `NULL`
+- `email` `NOT NULL`
+
+Now, let's suppose your application allow users to register just with their
```
Comment (mpavlisin) [inline README.md]: allows

File: README.md [hunk 6]
```diff
@@ -1,42 +1,131 @@
...
+`email` but can fill in they `firstname` and `lastname` later on, but we don't
```
Comment (mpavlisin) [inline README.md]: their

File: README.md [hunk 7]
```diff
@@ -1,42 +1,131 @@
...
+with your soft constraint.
+
+You can have as many soft constraints on as many servers and as many
+environments as you need.
+
+## Install as a cron
+
+If you want to get an email every night to give you a list of all the soft
+constraints that have been broken during the last day, just add it to you
+crontab. We like to have it run daily, so we can fix any bug generating bad
```
Comment (mpavlisin) [inline README.md:77]: We would like

**Comment by mpavlisin** [comment]
minor stuff but you can merge after 🚢

File: README.md (hunk 7 above)
Comment (lra) [inline README.md:77]: No, we actually like to have it run daily ;)

**Pull request merged**